distribution/errors.go

```diff
@@ -92,6 +92,9 @@ func retryOnError(err error) error {
 		case errcode.ErrorCodeUnauthorized, errcode.ErrorCodeUnsupported, errcode.ErrorCodeDenied:
 			return xfer.DoNotRetry{Err: err}
 		}
+		if strings.Contains(v.Error(), "too many failed login attempts for username or IP address") {
```

Comment:
Instinctively, I'd be a bit less specific, perhaps just check for "too many failed login attempts", but not sure if these messages ever change